### PR TITLE
feat: support for PR title prefix in upgrade action

### PR DESCRIPTION
**Pull request by jhdcruz**
Add support for adding **optional** prefixes in PR title.

Can be useful in a repo that utilizes conventional commits.

#### Sample Usage:
```yml
name: trunk
on:
  workflow_dispatch:

permissions: read-all

jobs:
  upgrade:
    name: upgrade trunk
    runs-on: ubuntu-latest
    permissions:
      contents: write
      pull-requests: write
    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Trunk Upgrade
        uses: trunk-io/trunk-action/upgrade@main
        with:
          prefix: "deps:"
```

### CODE REVIEW

File: upgrade/action.yaml [hunk 1]
```diff
@@ -97,14 +101,13 @@ runs:
...
-        title: |
-          ${{ env.PR_TITLE }}
+        title: ${{ inputs.prefix }} ${{ env.PR_TITLE }}
         body: ${{ env.PR_DESCRIPTION }}
         base: ${{ inputs.base }}
         branch: trunk-io/update-trunk
         labels: trunk
         add-paths: .trunk
-        commit-message: ${{ env.PR_TITLE }}
+        commit-message: ${{ inputs.prefix }} ${{ env.PR_TITLE }}
```
Comment (sxlijin) [inline upgrade/action.yaml:110]:
```suggestion
        commit-message: ${{ inputs.prefix }}${{ env.PR_TITLE }}
```

File: upgrade/action.yaml [hunk 2]
```diff
@@ -97,14 +101,13 @@ runs:
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@v5
       with:
-        title: |
-          ${{ env.PR_TITLE }}
+        title: ${{ inputs.prefix }} ${{ env.PR_TITLE }}
```
Comment (sxlijin) [inline upgrade/action.yaml:104]:
```suggestion
        title: ${{ inputs.prefix }}${{ env.PR_TITLE }}
```

**Review by sxlijin — COMMENTED** [review]
Thanks!

I take it that means this action is working for you then? :)

Also- thoughts on prefixing with `chore: ` vs `deps: `?

**Review by TylerJang27 — COMMENTED** [review]
LGTM, pending Sam's suggested changes. Thanks for the contribution!

**Comment by jhdcruz** [comment]
> I take it that means this action is working for you then? :)

Can confirm



> Also- thoughts on prefixing with `chore: ` vs `deps: `?

`deps: ` was just a sample usage I thought of at the moment of writing

**Review by sxlijin — APPROVED** [review]

**Comment by sxlijin** [comment]
Just a heads up - we're working on getting your PR merged. The "Trunk Check" status you see that's stuck in "Queued" is the result of some misconfiguration on our end.

**Comment by sxlijin** [comment]
Done- thanks for the contribution! Sorry it took so long.

**Pull request merged**